CHANGELOG.md

```diff
@@ -101,6 +102,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 | cadd        |             | 1.6.post1   |
 | gawk        |             | 5.3.0       |
 | echtvar     | 0.1.7       | 0.2.0       |
+| somalier    | 0.2.15      | 0.2.18      |
```

Comment:
only somalier that gets a new version?

Reply:
In this PR, yes, but I realised I have missed: 

`
add_most_severe_consequence: v1.0
add_most_severe_pli: v1.0
create_pedigree_file: v1.0
genmod=3.8.2
stranger:0.9.1
`

Will add these here.